dts/bindings/watchdog/microchip,xec-watchdog.yaml

```diff
@@ -9,19 +9,17 @@ title: Microchip XEC watchdog timer
 description: >
     This binding gives a base representation of the Microchip XEC watchdog timer
 
-inherits:
-    !include base.yaml
+include: base.yaml
 
-properties:
-    compatible:
-      constraint: "microchip,xec-watchdog"
+compatible: "microchip,xec-watchdog"
 
+properties:
     reg:
-      category: required
+      required: true
 
     label:
-      category: required
+      required: true
 
     interrupts:
-      category: required
+      required: true
 ...
```

Comment:
Can remove this document separator as well.

Reply:
Removed.